src/components/spec-document/samples/RequestSample.vue

```diff
@@ -267,8 +269,20 @@ watch(() => ({
     newValue.authHeaders !== oldValue?.authHeaders ||
     newValue.customHeaders !== oldValue?.customHeaders) {
 
+    let serverUrl: URL | undefined = undefined
+    const urlStr = (newValue.serverUrl + newValue.requestPath).replaceAll('{', '').replaceAll('}', '')
+    try {
+      serverUrl = new URL( urlStr )
```

Comment:
User can add their own custom URL, so we should reset the snippet error if the new serverUrl is valid

```suggestion
      serverUrl = new URL( urlStr )
      snippetError.value = false
```